include/ibamr/ConvectiveOperator.h

```diff
@@ -84,6 +84,11 @@ class ConvectiveOperator : public IBTK::GeneralOperator
      */
     virtual void applyConvectiveOperator(int Q_idx, int N_idx) = 0;
 
+    /*!
+     * \brief Compute N = div(Q1 Q2 u).
+     */
```

Comment:
Please make the variable name consistent between the comment and the function specification.